projects/packages/sync/src/class-listener.php

```diff
@@ -417,8 +417,16 @@ public function get_actor( $current_filter, $args ) {
 		);
 
 		if ( $this->should_send_user_data_with_actor( $current_filter ) ) {
-			require_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php';
-			$actor['ip']         = jetpack_protect_get_ip();
+			$ip = isset( $_SERVER['REMOTE_ADDR'] ) ? $_SERVER['REMOTE_ADDR'] : '';
```

Comment:
Would it make sense to simplify the complexity of the else { if () { statement into an else if { statement ?

} else if ( defined( 'JETPACK__PLUGIN_DIR' ) ) {

removes 2 addition lines and indenting.

Reply:
alternatively it could also be

if( function_exists( 'jetpack_protect_get_ip') || ( defined( 'JETPACK__PLUGIN_DIR' ) && include_once JETPACK__PLUGIN_DIR . 'modules/protect/shared-functions.php' ) ) {
    $ip = jetpack_protect_get_ip();
}


which would make it go from 8 lines to 3

Reply:
Great point! Updated the code.